tests/special_coverage/test_spec_cov_views.py

```diff
@@ -45,6 +45,11 @@ def test_special_coverage_view(self):
         self.assertEqual(response.context['special_coverage'], sc)
         self.assertEqual(response.context['content_list'].count(), sc.get_content().count())
         self.assertEqual(response.context['content_list'][0].id, content.id)
+        self.assertEqual(response.context['recirc'], None)
```

Comment:
@MichaelButkovic I'm not sure exactly what I should be testing for in the result here.

Should I make some content and check for it?

I'm not quite sure how to set that test up.